### ISSUE TITLE
Query from 2 fields

### ISSUE THREAD

**Issue by czkoudy** [opendns/autotask-php#132]
Hi,

Its me again. Is there a way I can pull all entities from TicketNote and TimeEntries for the same ticket ID in one request?

I need to be able to have a list of both ordered by date.

Any guidance would be helpful.

Thank you

Jakub

**Comment by SirPoot-** [comment]
No. You will have to make two requests and concatenate in your application.
